input_buffer.cc

```diff
@@ -874,6 +915,18 @@ expression_ptr text_input_buffer::parse_expression(bool stopAtParen)
 	source_location l = location();
 	switch (*(*this))
 	{
+		case '\'':
+			consume('\'');
+			if(!consume_char_literal(leftVal))
```

Comment:
if (

Reply:
@jlduran Do I need to engage?

Reply:
I'll submit a PR here addressing @jrtc27 style(9) changes.

(I was trying to add a super-linter to the project, to avoid this in the future, but it'll take a while).

Reply:
Done in #85

Will soon fix downstream (FreeBSD) as well. Thank you!